Assets/Plugins/LeapMotion/Core/Scripts/LeapServiceProvider.cs

```diff
@@ -534,8 +545,9 @@ public enum PhysicsExtrapolationMode {
     protected void onHandControllerConnect(object sender, LeapEventArgs args) {
       initializeFlags();
 
+      // TO REVIEW
       if (_leapController != null) {
-        _leapController.Device -= onHandControllerConnect;
+        // _leapController.Device -= onHandControllerConnect;
```

Comment:
Looking upstream from this event, it doesn't look like this causes any lasting issues when using a "Single-Device" Service, so I believe it's fine to leave in (pending a second opinion from @nickjbenson ).